staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -387,7 +387,8 @@ func (c *Config) Complete(informers informers.SharedInformerFactory) CompletedCo
 				},
 			}
 		}
-
+	}
+	if c.OpenAPIConfig != nil {
```

Comment:
sgtm. It would maybe clearer to keep everything in the `if c.OpenAPIConfig != nil` block, but guard only the above code with `c.OpenAPIConfig.SecurityDefinitions != nil`.

Reply:
either works for me.  I went for the route of "less nested indentation", but the other way seems clearer in other regards.  I'll switch.